### DISCUSSION TITLE
Issues with re-applying for GitHub Student Pack, despite providing all sorts of academic status proof with dates.

### DISCUSSION THREAD

**Discussion by JLG1995** [community/community#49882] [GitHub Education]
Hi, I've been trying to re-verify my my account for the Student pack(for a University course I am re-taking right now) through the upload academic status page but keep running into issues of it preventing me from doing so. I even took a picture of my course schedules with all the dates with my phone camera and it still keeps rejecting my re-application. Is there anything else that I might have forgotten or need to update on my account?
![Screenshot 2023-03-12 183132](https://user-images.githubusercontent.com/50001428/224588483-4f2abe58-f443-4c96-95ae-37a66c815217.jpg)

**Comment by bkpecho** [comment]
Hi @JLG1995,

Many people experience this issue, and to help you navigate it, I recommend referring to the [official documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) for clear instructions. 

If you continue to encounter difficulties, **consider completing** your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) by including your full name as it appears on your academic affiliation document, along with a brief bio. Doing so can significantly increase your chances of being verified.

Hope this helps and have a great day ahead.
